config/locales/es.yml

```diff
@@ -761,49 +760,49 @@ es:
       edit_13614c_form_page1:
         fields:
           apt: Apt.
-          below_qualifying_relative_income_requirement: "¿Tenía esta persona menos de $5,050 de ingresos?"
+          below_qualifying_relative_income_requirement: Did this person have less than $5,050 of income?
```

Comment:
should these be English? These are in the hub which doesn't technically support Spanish but the inconsistency seems weird